synapse/cmds/cortex.py

```diff
@@ -87,15 +87,37 @@ def nodevalu(t):
             return resp
 
         for node in nodes:
+
             form = node[1].get('tufo:form')
             valu = node[1].get(form)
 
-            tags = sorted(s_tufo.tags(node,leaf=True))
-            tags = [ '#'+tag for tag in tags ]
+            leafs = set(sorted(s_tufo.tags(node,leaf=True)))
```

Comment:
Agreed, sorted() should go ( it was leftovers from copy/pasta ).  The set() is being used to test individual tags for membership in the subset of "leaf" tags below.